src/orc.jl

```diff
@@ -0,0 +1,48 @@
+export OrcJIT, OrcModule, dispose, compile, address, addressin
+
+@checked struct OrcJIT 
+    ref::API.LLVMOrcJITStackRef
+end
+
+Base.unsafe_convert(::Type{API.LLVMOrcJITStackRef}, orc::OrcJIT) = orc.ref
+
+"""
+    ORC(::TargetMachine)
+
+Takes ownership of the provided target machine.
```

Comment:
Maybe better do this as a `!!! warning` or so, and use a regular description. Not that we have great documentation now, but we should start at some point :-)